### PR TITLE
[Task#79935] Make user CRUD function for admin

### PR DESCRIPTION
**Pull request by notverygood135**
https://edu-redmine.sun-asterisk.vn/issues/79935

### CODE REVIEW

**Comment by notverygood135** [comment]
ready

**Review by sonnguyendev157 — APPROVED** [review]

**Review by mvansan — APPROVED** [review]

File: ecommerce-app/app/Http/Controllers/Auth/RegisteredUserController.php [hunk 1]
```diff
@@ -30,16 +30,22 @@ public function create(): View
     public function store(Request $request): RedirectResponse
     {
         $request->validate([
```
Comment (ghost) [inline ecommerce-app/app/Http/Controllers/Auth/RegisteredUserController.php]: sử dụng https://laravel.com/docs/11.x/validation#form-request-validation để tách validate ra file riêng

File: ecommerce-app/app/Http/Controllers/UserController.php [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+     * Store a newly created resource in storage.
+     */
+    public function store(Request $request)
+    {
+        $user = new User;
+        $user->first_name = $request->first_name;
+        $user->last_name = $request->last_name;
+        $user->username = $request->username;
+        $user->email = $request->email;
+        $user->password = $request->password;
```
Comment (ghost) [inline ecommerce-app/app/Http/Controllers/UserController.php]: hash password

File: ecommerce-app/resources/css/app.css [hunk 3]
```diff
@@ -1,3 +1,46 @@
...
+  box-shadow: inset 0 0 0 9999px #111827;
+}
+
+.table > tbody {
+  vertical-align: inherit;
+}
+
+.table > thead {
+  vertical-align: bottom;
+}
```
Comment (ghost) [inline ecommerce-app/resources/css/app.css]: thêm dòng trống cuối file

**Review by ghost — APPROVED** [review]

**Pull request merged**